traitcuration/static/js/mapping.js

```diff
@@ -3,13 +3,21 @@ This module contains all the funcionality for mapping traits to ontology terms,
 */
 let selectedRowIndex = -1;
 let selectedTableId = "";
-let currentTraitId = -1;
+let currentTraitId = window.location.href.split("/").slice(-2)[0];
 let selectedTermId = -1;
 
-/* 
-This function takes in the clicked row of the mapping suggestion table, and the term and trait ids for its mapping
-suggestion, and sets them as selected
-*/
+
+// Check if the "Newly suggested terms" table is empty, and hide it if it is.
+const NewTermSuggestionTable = document.querySelector('#newSuggestionTable')
+console.log(NewTermSuggestionTable.rows.length)
+if (NewTermSuggestionTable.rows.length === 1) {
+  NewTermSuggestionTable.classList.add('hidden')
+  document.querySelector('#newSuggestionTable-title').classList.add('hidden')
+} 
```

Comment:
Wouldn't it possible to do it directly with Django while generating a page?

Reply:
I could pass a suggestion count in the page context during rendering, but I didn't want to make the view code too large. Should I do it this way?

Reply:
Oh, I just realised something. My original idea was to not include this table into a page _at all_ when there are no suggestions to show. But then obviously, new suggestions can be added by a user, so we still need this table to be present there even when it's empty.

So this was my mistake, and no action is required here.